docs/checks.md

```diff
@@ -12,13 +12,12 @@ If your PMM instance is not connected to Percona Platform, PMM can only download
 As soon as you connect your PMM instance to Percona Platform, has access to additional checks, available only for Registered PMM instances.
 
 If you are a Percona customer with a Percona Customer Portal account, you also get access to Paid checks, which offer more advanced database health information.
-​
 
 ## Checks results
 
-The checks can be executed manually or automatically. By default, PMM runs automatic checks every 24 hours.
+The checks can be executed manually or automatically. Manual checks can be executed individually, or all together. 
 
-You can configure this interval in **PMM > Advanced Settings > Execution intervals**. Here you can label each check individually as **Standard**, **Rare** or **Frequent** and also specify the time interval for each category.
+Automatic checks run every 24 hours, by default. You can configure this interval in **PMM > Advanced Settings > Execution intervals**. Here you can label each check individually as **Standard**, **Rare** or **Frequent** and also specify the time interval for each category.
```

Comment:
@atymchuk looks like we still mention it here. Though I guess we can clarify it further.

The "24 hours" interval is only true for a subset of checks (checks with a Standard interval). We have three intervals with these defaults:
Rare -> 78 hours
Standard -> 24 hours
Frequent -> 4 hours

So how often we run a check depends on the interval for that check.

Reply:
thanks, I've added this info too